### PR TITLE
Fix nil pointer and delete helmRelease 

### PR DESCRIPTION
**Pull request by xyz-li**
**What type of PR is this?**
1. fix nil pointer of attachment
2. delete helmRelease on host cluster when delete member cluster

/kind bug
/area app-management

/assign @zheng1 
/assign @zryfish

### CODE REVIEW

File: pkg/controller/openpitrix/helmrelease/helm_release_controller.go [hunk 1]
```diff
@@ -110,6 +113,24 @@ func (r *ReconcileHelmRelease) Reconcile(request reconcile.Request) (reconcile.R
 		// The object is not being deleted, so if it does not have our finalizer,
 		// then lets add the finalizer and update the object.
 		if !sliceutil.HasString(instance.ObjectMeta.Finalizers, HelmReleaseFinalizer) {
+			clusterName := instance.GetRlsCluster()
+			if clusterName != "" {
+				clusterInfo, err := r.clusterLister.Get(clusterName)
+				if err != nil {
+					if apierrors.IsNotFound(err) {
+						//cluster not exists, delete this helmRelease
```
Comment (LinuxSuRen) [inline pkg/controller/openpitrix/helmrelease/helm_release_controller.go:121]: Do nothing?

File: pkg/controller/openpitrix/helmrelease/helm_release_controller.go [hunk 2]
```diff
@@ -336,11 +358,22 @@ func (r *ReconcileHelmRelease) uninstallHelmRelease(rls *v1alpha1.HelmRelease) e
 	var clusterConfig string
 	var err error
 	if clusterName != "" && r.KsFactory != nil {
-		clusterConfig, err = r.clusterClients.GetClusterKubeconfig(clusterName)
+		clusterInfo, err := r.clusterLister.Get(clusterName)
 		if err != nil {
+			if apierrors.IsNotFound(err) {
+				klog.V(2).Infof("cluster %s is deleted, skip helm release uninstall", clusterName)
```
Comment (LinuxSuRen) [inline pkg/controller/openpitrix/helmrelease/helm_release_controller.go:364]: `is deleted` might be `was deleted`

File: pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go [hunk 3]
```diff
@@ -49,11 +54,11 @@ type releaseStatus struct {
 // Info describes release information.
 type Info struct {
 	// FirstDeployed is when the release was first deployed.
-	FirstDeployed time.Time `json:"first_deployed,omitempty"`
+	FirstDeployed helmtime.Time `json:"first_deployed,omitempty"`
```
Comment (LinuxSuRen) [inline pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go:57]: Please keep the mind the potential compatibility issues here.

File: pkg/controller/openpitrix/helmrelease/helm_release_controller.go (hunk 1 above)
Comment (xyz-li) [reply pkg/controller/openpitrix/helmrelease/helm_release_controller.go:121]: Yes.  If cluster was deleted, we just delete the crd and  left helm deployments on the member cluster

File: pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go (hunk 3 above)
Comment (xyz-li) [reply pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go]: I have removed this code.

File: pkg/controller/openpitrix/helmrelease/helm_release_controller.go (hunk 2 above)
Comment (xyz-li) [reply pkg/controller/openpitrix/helmrelease/helm_release_controller.go]: Got it.

**Comment by xyz-li** [comment]
Fix #3481 
Fix #3459

**Pull request merged**